### ISSUE TITLE
Sitemap fails Colour contrast

### ISSUE THREAD

**Issue by HerinHentry** [mr-bobz/talarts#120]
![image](https://cloud.githubusercontent.com/assets/15111528/11001074/6549102c-84f9-11e5-9f7d-9090a1f5860d.png)

**Comment by mr-bobz** [comment]
Should be fine now, pls recheck, thanks

**Comment by HerinHentry** [comment]
Excellent! Issue can be closed.
